compiler/rustc_mir_transform/src/lower_intrinsics.rs

```diff
@@ -333,6 +333,12 @@ impl<'tcx> MirPass<'tcx> for LowerIntrinsics {
                         });
                         terminator.kind = TerminatorKind::Goto { target };
                     }
+                    sym::contract_check => {
+                        if tcx.sess.opts.unstable_opts.contract_checking.no_checks() {
```

Comment:
the `no_checks` method tripped me up a bit. Slight preference for not having it at all and using an exhaustive match here instead